rental.rb

```diff
@@ -11,13 +11,4 @@ def initialize(date, person, book)
     @book = book
     book.rentals << self
   end
-
```

Comment:
Please add instructions to run the specs in your README file.

Reply:
Done, Thanks